src/aggregation/aggregator_wrapper.py

```diff
@@ -0,0 +1,19 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+class VideoCLIPAggregator(nn.Module):
+    """
+    Class to wrap aggregators so we can control their normalization and logit scale
+    """
+    def __init__(self, aggregator, logit_scale=100.0, normalize=True):
+        super().__init__()
+        self.aggregator = aggregator
+        self.logit_scale = logit_scale
+        self.normalize = normalize
+
+    def forward(self, x):
+        x = self.aggregator(x)
+        return self.logit_scale * F.normalize(
```

Comment:
do it like here https://github.com/mlfoundations/open_clip/blob/a6231f8cde726797330f149f1ffcfa0e63a55872/src/open_clip/model.py#L201

Reply:
as in return self.logit_scale.exp() and don't multiply it